packages/react-jss/src/with-styles.js

```diff
@@ -0,0 +1,168 @@
+// @flow
```

Comment:
Just realized you only renamed withStyles, not removed it. Please avoid making  changes which could be done in a separate step when needed, it makes it harder to review and discussion gets bloated with unrelated stuff. I would prefer to keep current conventions.

Reply:
For the sake of consistency with the rest.

Reply:
Done